libethereum/Executive.cpp

```diff
@@ -180,6 +180,11 @@ void Executive::initialize(Transaction const& _transaction)
 {
 	m_t = _transaction;
 
+	BlockHeader bh;
```

Comment:
sorry that was maybe already answered, but why do we need to call `verifyTransaction` at this point?